operations/image_management/Import_External_Image_to_IMS.md

```diff
@@ -84,45 +81,42 @@ the following procedure can be used to add this external image to IMS and upload
         ncn# cray artifacts create boot-images $IMS_IMAGE_ID/$IMS_ROOTFS_FILENAME $ROOTFS_FILENAME
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815